tests/integrations/flax/data_test.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict
+
+import jaxlib.xla_extension
+from datasets import load_dataset
+from transformers import AutoTokenizer
+
+from tango.integrations.flax import DataLoader, FlaxDataLoader
+from tango.integrations.flax.util import get_PRNGkey
+
+
+def test_dataloader() -> None:
+    assert "flax::dataloader" in DataLoader.list_available()
+
+
+def test_load_dataset() -> None:
+    dataset = load_dataset("glue", "mrpc", split="train")
+    tokenizer = AutoTokenizer.from_pretrained("bert-base-cased")
+    dataset = dataset.map(
+        lambda e: tokenizer(e["sentence1"], truncation=True, padding="max_length")
+    )
+    dataset = dataset.remove_columns(["sentence1", "sentence2", "idx"])
+    data = FlaxDataLoader(dataset, batch_size=16)
```

Comment:
In the jax codebase, is there a way to mock test distributed setting without actually having multiple devices? For example, in allennlp, we had this: https://github.com/allenai/allennlp/blob/main/allennlp/common/testing/distributed_test.py#L70 to simulate that setting.